test/fixtures/integration/extinf.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
       },
       duration: 10,
       timeline: 0,
-      uri: 'hls_450k_video.ts'
+      uri: 'hls_450k_video.ts',
+      title: ';asljasdfii11)))00,'
```

Comment:
Source text:

```
#EXTINF:;asljasdfii11)))00,
```

I think if we were to strictly follow the spec this should fail, since `;asljasdfii11)))00` is not a `decimal-floating-point` nor `decimal-integer`.
But I just went with the easiest route to make the tests pass.

Reply:
Oh, missed your comment, yeah, I think this is probably incorrect but that's likely an issue with the parsing/regex in https://github.com/videojs/m3u8-parser/blob/07bd668ddb5070fff57f5e5fe86959e18f49c4e3/src/parse-stream.js#L148-L162
and shouldn't be a blocker for this PR.